userspace/plugin/plugin_types.h

```diff
@@ -298,13 +298,13 @@ typedef enum ss_plugin_log_severity
 // Types supported by the by the metric values
 typedef enum ss_plugin_metric_value_type
 {
-	SS_PLUGIN_METRIC_VALUE_TYPE_U32 = 1,
-	SS_PLUGIN_METRIC_VALUE_TYPE_S32 = 2,
-	SS_PLUGIN_METRIC_VALUE_TYPE_U64 = 3,
-	SS_PLUGIN_METRIC_VALUE_TYPE_S64 = 4,
-	SS_PLUGIN_METRIC_VALUE_TYPE_D = 5,
-	SS_PLUGIN_METRIC_VALUE_TYPE_F = 6,
-	SS_PLUGIN_METRIC_VALUE_TYPE_I = 7,
+	SS_PLUGIN_METRIC_VALUE_TYPE_U32 = 0,
+	SS_PLUGIN_METRIC_VALUE_TYPE_S32 = 1,
+	SS_PLUGIN_METRIC_VALUE_TYPE_U64 = 2,
+	SS_PLUGIN_METRIC_VALUE_TYPE_S64 = 3,
+	SS_PLUGIN_METRIC_VALUE_TYPE_D = 4,
+	SS_PLUGIN_METRIC_VALUE_TYPE_F = 5,
+	SS_PLUGIN_METRIC_VALUE_TYPE_I = 6,
```

Comment:
Technically, this is a backward incompatible change, isn't it?

So, unfortunately, I believe we have to bump the `PLUGIN_API_VERSION_MAJOR` number.
cc @jasondellaluce 

/hold a bit for second opinions

Reply:
agree @leogr we can consider this feature not yet usable in the current release.

Reply:
Yet another option:
Forcing the enum values in `metrics_v2.h` (https://github.com/falcosecurity/libs/blob/master/userspace/libscap/metrics_v2.h) to match the enums in `plugin_types.h`
This should work given that these enums are only casted here https://github.com/falcosecurity/libs/blob/master/userspace/libsinsp/plugin.cpp#L947-L948

Reply:
Agree with Leo, since it was unusable in 0.17.1, we can consider a minor bump.